### DISCUSSION TITLE
VSCode Cycling between suggestions doesn't work

### DISCUSSION THREAD

**Discussion by rastik1337** [community/community#7401] [Copilot Conversations]
<s>I'm unable to cycle between suggestions even If there is more of them, the **Next** and **Previous** are just unavailable.
Everything else works for me. (i.e: **CTRL + Enter** to Open Github Copilot etc..)
</s>
Editor: VSCode
![image](https://user-images.githubusercontent.com/38767414/140615844-d614a8c2-83b5-4337-a7ef-bbb4da1618bb.png)

Edit:
It just seems like there can be more suggestions, but there is just one. **It works as it should.**

**Comment by Johanneshn** [comment]
I'm having the same issue, using v1.7.3689 in typescript. Copilot is the only extension running.
ctrl+enter gives me multiple solutions, but Next/Previous is grayed out in the hover dialog and keyboard shortcuts don't work either.

![image](https://user-images.githubusercontent.com/3512547/140712669-db99de7a-f715-4bf6-98cc-c863ccd00f8d.png)

**Comment by marlonfan** [comment]
My vscode suggestion list conflicts with code hints using v1.7.3689

**Comment by chriszrc** [comment]
Same here. I was hoping that this would be as simple as choosing new key bindings, but this is what vs code shows for copilot key bindings:

<img width="1097" alt="Screen Shot 2022-03-08 at 11 07 54 AM" src="https://user-images.githubusercontent.com/25695785/157265764-4a3e34f8-950d-456f-9890-07342ee87223.png">

No next or previous anywhere :(

**Reply by supersurviveur** [reply]
You can change key bindings for next and previous with
`editor.action.inlineSuggest.showPrevious` and `editor.action.inlineSuggest.showNext`
![Capture d’écran du 2022-03-30 17-30-28](https://user-images.githubusercontent.com/54775010/160873465-ef203a56-5d0d-4e62-9542-7cb88fa82106.png)

https://github.com/github/copilot-docs/blob/main/docs/visualstudiocode/configuring.md

**Comment by mCr86** [comment]
Same issue for me.
Windows 10 - vscode 1.66.0
![image](https://user-images.githubusercontent.com/49694186/161238824-1f3a868b-ae8c-4479-acbd-6af9502da3bf.png)
I tried to change the shortcuts, but the result is the same.

**Comment by N4SIRODDIN3** [comment]
I think that is because no more suggestions could be proposed by the copilot.
Try this one in python and see if you could toggle among suggestions:
```py
# solve a quadratic equation ax^2 + bx + c = 0
def qua
```

**Reply by Vrajs16** [reply]
Thanks, cause I thought it was broken and always had to do ```^ Enter```

**Comment by stoplion** [comment]
Suggestion keybinding not working in VSCode.

<img width="406" alt="image" src="https://user-images.githubusercontent.com/231343/167500892-3fdbfeb6-12f6-4c01-9684-8607c8a5c6d3.png">

I've disabled all extensions except for CoPilot.

`^[` and `^]` are assigned to Copilot suggestions.

Those above keybinding are not cycling suggestions..

Also, when I open Copilot 10/10 solutions, all are hidden
<img width="542" alt="image" src="https://user-images.githubusercontent.com/231343/167502800-50542134-fa1b-4f2c-a70e-16272820a94b.png">

**Comment by caboe** [comment]
Same here. Used the example from https://docs.github.com/en/copilot/getting-started-with-github-copilot/getting-started-with-github-copilot-in-visual-studio-code and getting a suggestion. Hitting Alt-Enter also shows that there are some solutions available. But the shortcuts for selecting next/previous are disabled, as you can see in the popover:
<img width="845" alt="Bildschirmfoto 2022-08-21 um 14 42 10" src="https://user-images.githubusercontent.com/627518/185791514-2e1a3b6c-698c-4069-ac57-7a83ee3f2d6c.png">
